# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1553048733 0
#      Wed Mar 20 02:25:33 2019 +0000
# Node ID 05db503583771883ac5b21aa23cc1e8ef63b4404
# Parent  791cb021ccb89489d7a087cc24882fb257170a00
* update-bug-status.sh: Delete newline chars in wget output

diff --git a/build-aux/update-bug-status.sh b/build-aux/update-bug-status.sh
--- a/build-aux/update-bug-status.sh
+++ b/build-aux/update-bug-status.sh
@@ -18,17 +18,17 @@
 # along with Octave; see the file COPYING.  If not, see
 # <https://www.gnu.org/licenses/>.
 
 bug_numbers=$(for file in "$@"; do
   sed -n "s/.*<\([0-9][0-9][0-9][0-9]*\)>.*/\1/p" "$file"
 done | sort -u)
 
 fixed_bug_numbers=$(for num in $bug_numbers; do
-  status=$(wget -q -O - https://octave.org/testfailure/?$num | sed -n 's/.*>Status:<\/span><\/span>&nbsp;<\/td><td valign="middle" width="35%">\([^<]*\)<.*/\1/p');
+  status=$(wget -q -O - https://octave.org/testfailure/?$num | tr -d '\n' | sed -n 's/.*>Status:<\/span><\/span>&nbsp;<\/td> *<td valign="middle" width="35%">\([^<]*\)<.*/\1/p');
   if [ "$status" = "Fixed" ]; then echo "$num"; fi
 done)
 
 if [ -z "$fixed_bug_numbers" ]; then
   echo "no change in bug status"
   exit 0;
 fi
 
